src/game/GameScene.ts

```diff
@@ -0,0 +1,32 @@
+import { KaboomCtx, LevelOpt } from "kaplay";
+import GameInstance from "./Game";
+import Player from "../components/Player";
+
+export class GameScene {
```

Comment:
you can move the scene to the Game class, hence making a scene from the Game itself, which makes a lot of sense.